pkg/storaged/nfs-details.jsx

```diff
@@ -17,6 +17,8 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 
+import "polyfills";
```

Comment:
Oh cool - other pages use `import '../lib/polyfills.js'`, if that shorter form works we should use that consistently; looks so much nicer :-)   (unrelated to this PR of course)

Reply:
Yes, `pkg/lib` is in the search path: https://github.com/cockpit-project/cockpit/blob/master/webpack.config.js#L407